Products/DataCollector/SshClient.py

```diff
@@ -761,15 +762,15 @@ class SshClient(CollectorClient.CollectorClient):
     """
 
     def __init__(
-        self,
-        hostname,
-        ip,
-        port=22,
-        plugins=[],
-        options=None,
-        device=None,
-        datacollector=None,
-        isLoseConnection=False,
+            self,
+            hostname,
+            ip,
+            port=22,
+            plugins=[],
+            options=None,
+            device=None,
+            datacollector=None,
+            isLoseConnection=False,
```

Comment:
No need to reformat: single indent is preferred over double indent.

Reply:
I found in [Code Standards](https://zenoss.atlassian.net/wiki/spaces/SrvDev/pages/648478724/Code+Standards+Best+Practices) that we should use 4 space indent, i.e. double indent. I thought we should follow the standards.

Reply:
I used PyCherm Reformat Code.